bower.json

```diff
@@ -19,14 +19,14 @@
     "paper-elements": "polymerelements/paper-elements#^1.0.7",
     "carbon-elements": "polymerelements/carbon-elements#^0.1.0",
     "iron-elements": "polymerelements/iron-elements#^1.0.10",
-    "carbon-route": "TimvdLippe/carbon-route#subroute-updating",
+    "carbon-route": "TimvdLippe/carbon-route#reset-tail",
```

Comment:
I updated my original PR (`#subroute-updating`) with some fixes. However, there was some discussion for 2 changes so I did not include these in the PR. Instead, I branched of into `#reset-tail` until further discussion has been resolved. For reference see https://github.com/PolymerElements/carbon-route/pull/77#discussion_r63082376